seed/seed.py

```diff
@@ -0,0 +1,25 @@
+import psycopg2
+
+conn = psycopg2.connect(
```

Comment:
These values definitely shouldn't be hard coded. Use the URI string constant in `config.py` or if that's not possible, get the values from the environment file.

Reply:
updated